src/figuresClasses.ts

```diff
@@ -1,19 +1,94 @@
-export interface Figure {
+enum Shape {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle',
+}
 
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
 }
 
-export class Triangle {
+const roundtoHundredths = (num: number): number => Math.floor(num * 100) / 100;
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  a?: number;
+  b?: number;
+  c?: number;
+  radius?: number;
+  width?: number;
+  height?: number;
+  getArea(): number;
 }
```

Comment:
the Figure interface should contain only 3 properties
![image](https://user-images.githubusercontent.com/98764641/213429438-16dcfbf1-ab2c-4739-9543-ac6ac9170490.png)
other will be added to each child's class